front/src/modules/simulationResult/components/ChartHelpers/__tests__/ChartHelpers.spec.ts

```diff
@@ -1,5 +1,4 @@
 import { LIST_VALUES, CHART_AXES } from 'modules/simulationResult/consts';
-import type { SimulationTrain } from 'reducers/simulationResults/types';
 
 import simulationTrain from '../../../../../../tests/assets/operationStudies/simulationTrain';
```

Comment:
Could you fix these ugly imports ? :)

Reply:
I think I can't since this file is outside the frontend folder :/